### PR TITLE
JsonDecodeException

### PR DESCRIPTION
**Pull request by psfpro**
This will be easier to debug
```php
        try {
            $response = $this->client->send($request);
        } catch (JsonDecodeException $exception) {
            $this->log($exception->getMessage(), $exception->getJson());
        }
```

### CODE REVIEW

File: src/Exception/JsonDecodeException.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    /**
+     * @return string
+     */
+    public function getJson()
+    {
+        return $this->json;
+    }
+}
```
Comment (h-bragg) [inline src/Exception/JsonDecodeException.php]: [phpcs] Missing new line

File: src/Exception/JsonDecodeException.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+use Throwable;
+
+class JsonDecodeException extends \InvalidArgumentException
+{
+    /**
+     * @var string
+     */
+    private $json;
+
+    public function __construct($message = "", $code = 0, Throwable $previous = null, $json)
```
Comment (h-bragg) [inline src/Exception/JsonDecodeException.php]: [phpcs] Missing doc block

**Comment by psfpro** [comment]
Hi! Coding standards fixed

**Comment by h-bragg** [comment]
Hi, thanks for the fixes 👏.

**Pull request merged**